exercises/practice/robot-name/.approaches/introduction.md

```diff
@@ -31,8 +31,7 @@ There's some basic stats to help you grasp the issue:
     Probability of finding the last name with N guesses =
       1 - ((676000 - 1) / 676000)^N
 
-
-## A fancy solution
+## Approach: shuffle all robot names in advance
```

Comment:
```suggestion
## Approach: encode names as numbers and shuffle them all in advance
```

Encoding names as numbers is the trickiest part of this, so let's lede (geddit) with that